### PR TITLE
Add 1 to phone numbers

### PR DESCRIPTION
**Pull request by robbiethegeek-usds**

### CODE REVIEW

**Review by bethpotts — APPROVED** [review]
Looks good! Thanks so much, Robbie.

File: src/js/edu-benefits/1990-rjsf/containers/ConfirmationPage.jsx [hunk 1]
```diff
@@ -127,7 +127,7 @@ class ConfirmationPage extends React.Component {
           </ul>
         </div>
         <p>Find out what happens <a href="/education/after-you-apply">after you apply</a>.</p>
-        <p>Need help? If you have questions, call <a href="tel:888-442-4551">888-442-4551</a> (888-GI-BILL-1) from 8:00 a.m. - 7:00 p.m. ET Mon - Fri.</p>
+        <p>Need help? If you have questions, call <a href="tel:888-442-4551">1-888-442-4551</a> (888-GI-BILL-1) from 8:00 a.m. - 7:00 p.m. ET Mon - Fri.</p>
```
Comment (jbalboni) [inline src/js/edu-benefits/1990-rjsf/containers/ConfirmationPage.jsx]: We don't need to add `1` to the `888-GI-BILL-1` version?

**Review by jbalboni — APPROVED** [review]

**Comment by jbalboni** [comment]
I think the build failure will be resolved by merging the latest from master.

File: src/js/edu-benefits/1990-rjsf/containers/ConfirmationPage.jsx (hunk 1 above)
Comment (bethpotts) [reply src/js/edu-benefits/1990-rjsf/containers/ConfirmationPage.jsx]: @jbalboni The 1 should be added to all phone numbers. Looks like you caught one where it's missing?  Can you add it?

Comment (robbiethegeek-usds) [reply src/js/edu-benefits/1990-rjsf/containers/ConfirmationPage.jsx]: Updated.  Didn't add it cause it is a vanity phone number.

**Pull request merged**